soh/soh/SohGui/SohMenuEnhancements.cpp

```diff
@@ -65,6 +65,7 @@ void SohMenu::AddMenuEnhancements() {
             }
             CVarSetInteger(presetTypeCvar.c_str(), selectedPresetId);
             Ship::Context::GetInstance()->GetWindow()->GetGui()->SaveConsoleVariablesNextFrame();
+            ShipInit::Init("*");
```

Comment:
Can we get this inside applyPreset() so we don't need to put them in individual places? Will be easier to track when we implement better presets too.

Reply:
Fixed in 816ef90